projects/laji/src/app/shared-modules/spreadsheet/importer/status-cell/error-list/error-list.component.ts

```diff
@@ -84,13 +84,8 @@ export class ErrorListComponent {
       return this.translateService.instant('haseka.form.genericError');
     }
     let base = err.message;
-    if (Util.hasOwnProperty(err, 'params') && typeof err.params === 'object') {
-      const info = [];
-      Object.keys(err.params).forEach(key => {
-        if (typeof err.params[key] === 'string') {
-          info.push(err.params[key]);
-        }
-      });
+    if (Util.hasOwnProperty(err, 'params') && typeof err.params === 'object' && err.params && Array.isArray(err.params)) {
```

Comment:
Did you mean `Array.isArray(err.params)`? Now you are checking for an array, previously it seemed that you were checking for an object here.